src/modules/internet/index.ts

```diff
@@ -122,14 +123,18 @@ export class InternetModule {
   }
 
   /**
-   * Generates a username using the given person's name as base. This will always return a plain ASCII string.
+   * Generates a username using the given person's name as base. This will always return a plain ASCII string. Some basic stripping of accents and transliteration of characters will be done.
    *
    * @param firstName The optional first name to use. If not specified, a random one will be chosen.
-   * @param lastName The optional last name to use. If not specified, a random one will be chosen.
+   * @param lastName The optional last name to use - though the last name is not always used. If not specified, a random one will be chosen.
    *
    * @example
    * faker.internet.userName() // 'Nettie_Zboncak40'
    * faker.internet.userName('Jeanne', 'Doe') // 'Jeanne98'
+   * faker.internet.userName('John', 'Doe') // 'John.Doe'
+   * faker.internet.userName('Hélene', 'Müller') // 'Helene_Muller11'
+   * faker.internet.userName('Фёдор', 'Достоевский') // 'Fedor.Dostoevskii50'
+   * faker.internet.userName("大羽","陳") //hlzp8d.tpv45
```

Comment:
All the other examples use single quotes:

```suggestion
   * faker.internet.userName('大羽', '陳') // 'hlzp8d.tpv45'
```